package.xml

```diff
@@ -17,6 +17,7 @@
   <author>Ken Conley</author>
 
   <buildtool_depend version_gte="0.5.74">catkin</buildtool_depend>
+  <run_depend version_gte="0.5.74">catkin</run_depend>
```

Comment:
Yes, the minimum version for the build dependency was introduced in 6bbac5331acb6673e361621609de11e1e3c70071 to ensure a new enough catkin version is being used when compiling the package. That is not necessary for the runtime of the package.